### ISSUE TITLE
Could this be used to create a chatserver?

### ISSUE THREAD

**Issue by jhjacobs81** [gbaranski/ezsockets#4]
Where users log in, join channels, and send texts? And if so, do you have any pointers? I’m looking to learn rust :)

**Comment by gbaranski** [comment]
Yes it is, at the moment you can take a look at https://github.com/gbaranski/ezsockets/tree/master/examples/chat-server, but it doesn't support rooms yet, I'll try to add them by this weekend.

**Comment by gbaranski** [comment]
I’ve just added more complex version of chat server, you can take a look at it https://github.com/gbaranski/ezsockets/tree/master/examples/chat-server

**Comment by jhjacobs81** [comment]
Woah :) you are way ahead of me :)
